src/drivers/distance_sensor/sf1xx/sf1xx.cpp

```diff
@@ -857,13 +858,32 @@ info()
 static void
 sf1xx_usage()
 {
-	PX4_INFO("usage: sf1xx command [options]");
-	PX4_INFO("options:");
-	PX4_INFO("\t-b --bus i2cbus (%d)", SF1XX_BUS_DEFAULT);
-	PX4_INFO("\t-a --all");
-	PX4_INFO("\t-R --rotation (%d)", distance_sensor_s::ROTATION_DOWNWARD_FACING);
-	PX4_INFO("command:");
-	PX4_INFO("\tstart|stop|test|reset|info");
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
+### Description
+
+I2C bus driver for Lightware SFxx series LIDAR rangefinders: SF10/a, SF10/b, SF10/c, SF11/c, SF/LW20.
+
+Setup/usage information: https://docs.px4.io/en/sensor/sfxx_lidar.html
+
+### Examples
+
+Attempt to start driver on any bus (start on bus where first sensor found).
+$ sf1xx start -a
+Stop driver
+$ sf1xx stop
+)DESCR_STR");
+
+	PRINT_MODULE_USAGE_NAME("sf1xx", "driver");
+	PRINT_MODULE_USAGE_COMMAND_DESCR("start","Start driver");
+	PRINT_MODULE_USAGE_PARAM_FLAG('a', "Attempt to start driver on all I2C buses ([--all])", true);
+	PRINT_MODULE_USAGE_PARAM_INT('b', 1, 1, 2000, "Start driver on specific I2C bus ([--bus])", true);
```

Comment:
How are I2C bus mapped/do they work. I.e. can I assume that every board will have at least two I2C bus, numbered 1, 2, now and forevermore? Do people think of these bus as "1" and "2" or would a developer have another idea of them as a "handle"?

Reply:
Yes they are numbered. It's board-specific which ones are internally only, and which ones are externally. This should be part of the board documentation.